ui/dasher/src/piece.ts

```diff
@@ -53,17 +58,18 @@ function pieceImage(t: Piece, is3d: boolean) {
   return `piece/${t}/wN.svg`;
 }
 
-const pieceView = (current: Piece, set: (t: Piece) => void, is3d: boolean) => (t: Piece) =>
-  h(
+const pieceView = (current: Piece, set: (t: Piece) => void, is3d: boolean) => (t: Piece) => {
+  const pieceSize = (225 - elementScrollBarWidth()) / 3;
```

Comment:
hardcoded pixel width from the CSS + the `elementScrollBarWidth` hack. And the result is still broken
![SJ4hLHw](https://github.com/lichess-org/lila/assets/140370/6e8fbdfd-1d5a-468c-bb18-d42e23dd41d4)

Is there another way?